src/server/src/main/resources/db/astra/001_init_reaper.cql

```diff
@@ -0,0 +1,171 @@
+--
+--  Copyright 2020 DataStax, Inc.
+--
+--  Licensed under the Apache License, Version 2.0 (the "License");
+--  you may not use this file except in compliance with the License.
+--  You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+--  Unless required by applicable law or agreed to in writing, software
+--  distributed under the License is distributed on an "AS IS" BASIS,
+--  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+--  See the License for the specific language governing permissions and
+--  limitations under the License.
+--
+
+CREATE TABLE IF NOT EXISTS repair_schedule_by_cluster_and_keyspace (
```

Comment:
what's different about the astra migration files? arn't all cassandra backends the same?

Reply:
Nope, there are many table settings that we can't use, such as compaction strategies.
So we need to have specific create table statements without table settings.

Reply:
Sample output from Astra: 

```
reaper@cqlsh:test> create table test(id int PRIMARY KEY, val text) WITH compaction = {'class':'LeveledCompactionStrategy'};
InvalidRequest: Error from server: code=2200 [Invalid query] message="Provided values [compaction] are not allowed for Table Properties (disallowed values are: [additional_write_policy, bloom_filter_fp_chance, caching, cdc, comment, compaction, compression, crc_check_chance, dclocal_read_repair_chance, extensions, gc_grace_seconds, id, max_index_interval, memtable_flush_period_in_ms, min_index_interval, nodesync, read_repair, read_repair_chance, speculative_retry])"
```